### PR TITLE
upgrade product chart version

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFR-1251

### Change description ###
Upgrade product chart versions for aac & xui 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -106,9 +106,9 @@ withPipeline("nodejs", product, component) {
   before('functionalTest:preview') {
     yarnBuilder.yarn('setup')
 
-    env.COS_API_URL="http://cos-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
-    env.AAC_API_URL="http://aac-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
-    env.CCD_DATA_API_URL="http://ccd-data-store-api-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.COS_API_URL="https://cos-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
```
Comment (adusumillipraveen) [inline Jenkinsfile_CNP]:
```suggestion
    env.COS_API_URL="http://finrem-ccd-definitions-pr-830-finrem-cos"
```

Comment (adusumillipraveen) [reply Jenkinsfile_CNP]: This seems to be a URL used for svc to svc communication, it has to be svc url

Comment (rishikrsharma) [reply Jenkinsfile_CNP]: applied change

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -177,7 +177,7 @@ withPipeline("nodejs", product, component) {
 
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'definitions/**/xlsx/*.xlsx'
     env.CCD_ADMIN_URL="https://ccd-admin-web.aat.platform.hmcts.net"
-    env.CCD_DATA_API_URL="http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+    env.CCD_DATA_API_URL="https://ccd-data-store-api-aat.service.core-compute-aat.internal"
```
Comment (adusumillipraveen) [inline Jenkinsfile_CNP:180]: AAT URLs are still http
```suggestion
    env.CCD_DATA_API_URL="http://ccd-data-store-api-aat.service.core-compute-aat.internal"
```

File: Jenkinsfile_nightly [hunk 3]
```diff
@@ -41,7 +41,7 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 
 withNightlyPipeline("nodejs", product, component) {
     env.CCD_WEB_URL="https://manage-case.aat.platform.hmcts.net"
-    env.CCD_DATA_API_URL="http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+    env.CCD_DATA_API_URL="https://ccd-data-store-api-aat.service.core-compute-aat.internal"
```
Comment (adusumillipraveen) [inline Jenkinsfile_nightly:44]: AAT URLs should be untouched.
```suggestion
    env.CCD_DATA_API_URL="http://ccd-data-store-api-aat.service.core-compute-aat.internal"
```

File: Jenkinsfile_CNP (hunk 2 above)
Comment (rishikrsharma) [reply Jenkinsfile_CNP]: change applied

File: Jenkinsfile_nightly (hunk 3 above)
Comment (rishikrsharma) [reply Jenkinsfile_nightly]: applied change

File: Jenkinsfile_CNP [hunk 4]
```diff
@@ -106,9 +106,9 @@ withPipeline("nodejs", product, component) {
   before('functionalTest:preview') {
     yarnBuilder.yarn('setup')
 
-    env.COS_API_URL="http://cos-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
-    env.AAC_API_URL="http://aac-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
-    env.CCD_DATA_API_URL="http://ccd-data-store-api-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.COS_API_URL="http://finrem-ccd-definitions-pr-${CHANGE_ID}-finrem-cos"
+    env.AAC_API_URL="https://aac-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
```
Comment (adusumillipraveen) [inline Jenkinsfile_CNP:110]: these 2 might need a change too?

**Review by EllisD-B — APPROVED** [review]

File: Jenkinsfile_CNP (hunk 4 above)
Comment (rishikrsharma) [reply Jenkinsfile_CNP:110]: looking this - making change

**Review by BWeeks-JusticeGovUK — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**